backend/src/GraphQl/OpenIdConnect/Application/ApplicationType.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using HotChocolate.Types;
+using OpenIddict.EntityFrameworkCore.Models;
+
+namespace Metabase.GraphQl.OpenIdConnect.Application;
+
+public sealed class ApplicationType
+    : ObjectType<OpenIddictEntityFrameworkCoreApplication>
+{
+    protected override void Configure(
+        IObjectTypeDescriptor<OpenIddictEntityFrameworkCoreApplication> descriptor
+    )
+    {
+        const string suffixedName = nameof(ApplicationType);
+        descriptor.Name(suffixedName.Remove(suffixedName.Length - "Type".Length));
+
+        descriptor.Field(t => t.Permissions).Type<NonNullType<ListType<StringType>>>().Resolve(context =>
+        {
+            var application = context.Parent<OpenIddictEntityFrameworkCoreApplication>();
+            var permissions = JsonSerializer.Deserialize<List<string>>(application.Permissions!);
+            return permissions?.FindAll(permission => permission.Contains("api")).ToList();
```

Comment:
The string `"api"` is also used above. Could you turn it into a constant and use the constant instead at all places in which you use `"api"` right now. I would put it in `AuthConfiguration` before `ReadApiScope` and so forth and if possible construct the constant `ReadApiScope` and the others from the new constant.